components/raftstore/src/router.rs

```diff
@@ -47,17 +48,13 @@ where
     }
 
     /// Sends RaftCmdRequest to local store.
-    fn send_command(&self, req: RaftCmdRequest, cb: Callback<EK::Snapshot>) -> RaftStoreResult<()> {
```

Comment:
Why not keep it? So most places are necessary to change. We just need to change `send_command_with_deadline` to `send_command_with_options`.